libcodechecker/server/database/database_status.py

```diff
@@ -0,0 +1,24 @@
+
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+"""
+Map between database statuses and the corresponding messages.
+"""
+from shared.ttypes import DBStatus
+
+db_status_msg = {
+    DBStatus.OK: "Database is up to date.",
+    DBStatus.MISSING: "Missing database.",
+    DBStatus.FAILED_TO_CONNECT: "Connection failed.",
```

Comment:
I think to emphasize the difference between this and the previous, we should reword some keys here: `Connection failed to server.`.